cms/envs/common.py

```diff
@@ -944,6 +944,10 @@
 
     # management of user-triggered async tasks (course import/export, etc.)
     'user_tasks',
+
+    # CMS specific user task handling
+    'cms_user_tasks',
```

Comment:
If you don't reference `CmsUserTasksConfig` here, there should be a `default_app_config` in the module's `__init__.py` pointing to it.